### ISSUE TITLE
Slangroom chain data transformation can not be used

### ISSUE THREAD

**Issue by matteo-cristino** [ForkbombEu/ncr#177]
The step functionality:
* `dataTransform`
* `keysTransform`
* `onAfter`
* `onBefore`

can not be used since they are a js function an when present ncr throws the error:
```
node:internal/process/promises:288
            triggerUncaughtException(err, true /* fromPromise */);
            ^

Error: /verify.chain.json: malformed JSON
    at Directory.getJSON (/snapshot/ncr/dist/index.cjs)
    at /snapshot/ncr/dist/index.cjs
    at Map.forEach (<anonymous>)
    at get files [as files] (/snapshot/ncr/dist/index.cjs)
    at generateRoutes (/snapshot/ncr/dist/index.cjs)
    at LiveDirectory.<anonymous> (/snapshot/ncr/dist/index.cjs)

Node.js v18.5.0
```
when trying to read the chain as json.

**Comment by puria** [comment]
We just decided togethre that this can be mitigated by allow to have .js files in place of .json

**Comment by matteo-cristino** [comment]
Solved in #195 by making the chain a js file and using the `eval` function when reading it
